salt/docker/init.sls

```diff
@@ -98,3 +98,7 @@ docker-reload-config:
     - reload: True
     - onchanges:
       - file: /etc/docker/daemon.json
+  {% for cert_tuple in certs.items() %}
+    {% set name, _cert = cert_tuple %}
+      - file: /etc/docker/certs.d/{{ name }}/ca.crt
```

Comment:
@inercia btw i highly suspect that this is also reloading on each cert change...

Reply:
Please make sure if that is the case and we will report it upstream to the Salt developers.

Reply:
i'll have a look upstream. can you give your last review on this one?